### PR TITLE
Added method rep()

### LINKED ISSUE #2: New Method: rep(arg1, arg2,..., argN, howManyTimes)
This method concatenates the same string a given number of times.
```js
var sb = new StringBuilder(); 
sb.cat(‘Mom, can you’).rep(‘ please’, 10).cat(‘ buy me an ice cream’);
```
As you can see this method can be very useful to annoy your mom.  It can also be combined with cat using the cascade or chaining pattern (remember it, because it’s the last time I say it).

### PR DESCRIPTION
**Pull request by islas27**
- Added documentation
- Added unit test

Branch based on feature/method-cat 5ea3ee1 and closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,77 @@
...
+**Example**: [cat.js](examples/cat.js)
+
+**Notes**:
+- When `typeof` of an input evaluates to `object` or `undefined`, this argument will be ignored. This happens to any kind of object, map, null or undefined values. To know more about the `typeof` operand, follow this link: [typeof - MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof)
+- While arrow functions maybe more clean and concise, the do not bind the `this` object, so in case you need to use the helper inside a function that is going to be passed as an argument, write an old fashion function. More info on arrow functions: [Arrow Functions - MDN](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
+
+---
+
+### rep(str, repetitions)
+The functions will append at the end of the current buffer the string sent the number of times indicates.
```
Comment (ryanez) [inline README.md]: Mi no entender, what did you say?
In the other hand this is not what the requirements says about `rep` method. I'm quite sure I saw a lot more of arguments on the drive document.

File: test/rep.spec.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+const expect = require('chai').expect
+
+describe('#rep', () => {
+  it('appends by default one time the string if no `repetitions` is sent', () => {
+    let helper = new StringHelper('ones: ')
+    let result = helper.rep('1').str()
+    expect(result).to.equal('ones: 1')
+  })
+
+  it('convertes to a number 1 the `rep` argument if not a number', () => {
```
Comment (ryanez) [inline test/rep.spec.js]: If you mean 'converts' you have an extra 'e' but if you meant 'convierte' your are missing an 'i' and still having an extra 's', whatever the case is, you know what I mean.

File: README.md (hunk 1 above)
Comment (islas27) [reply README.md]: Yeah, my bad. I miss understood the problem. Im working on it!

**Comment by islas27** [comment]
Alright, here's the fix! @ryanez

File: index.js [hunk 3]
```diff
@@ -10,10 +11,9 @@ function stringHelper () {
     const len = arguments.length
     for (let i = 0; i < len; i += 1) {
       let arg = arguments[i]
-      let type = classifier(arg)
-      if (type === 'function') this.cat(arg.apply(this))
-      else if (type === 'array') this.cat(...arg.flatten())
-      else if (type === 'undefined' || type === 'object') continue
+      if (isType('function', arg)) this.cat(arg.apply(this))
```
Comment (ryanez) [inline index.js]: using string literals among all the code is not a good practice, what about having something more semantic like
`isBoolean(value)` the linter will tell you about not defined `isBoolean` function but not that `isType('buulean', value)` will never match anything.

Comment (ryanez) [reply index.js]: think about the `isNaN` function 😉

Comment (islas27) [reply index.js]: Then, if Im almost always checking if something is a function, array or boolean, should I have 3 functions (isBool, isFunc, isArr)?

Comment (islas27) [reply index.js]: BTW, tried to use `!isNaN()`, but it broke the behaviour, as a `'1'` evaluates to true, while I believe it should be treated as a string to be appended. As such, I made a `isNum()` function using `typeof`. Following this logic, made helper functions for function, array, number and elements to be ignored (like undefined and objects). Commit 94a43da

**Comment by ryanez** [comment]
LGTM 👍

**Pull request merged**